processing/src/main/java/org/apache/druid/segment/data/ColumnarDoublesSerializer.java

```diff
@@ -29,5 +29,6 @@
 public interface ColumnarDoublesSerializer extends Serializer
 {
   void open() throws IOException;
+  int size();
```

Comment:
javadocs please. What does the size indicate? Number of rows in the column? Or space it's taking up. Reading the code it looks like it's the former

Reply:
aw, I was just making it symmetrical with [ColumnarFloatsSerializer](https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/segment/data/ColumnarFloatsSerializer.java) and [ColumnarLongsSerializer](https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/segment/data/ColumnarLongsSerializer.java), neither of which have any docs either. 

This area of the code in general is sort of barren of javadocs, so how about a bargain: unless there is something else to change on this PR, how about I just do a follow-up that adds javadocs to a bunch of this stuff?

Reply:
sounds like a good deal to me. I'm almost done reviewing, pressed the wrong button when I made that comment. I don't have anything else yet.